src/pas/plugins/headers/tests/test_parsers.py

```diff
@@ -18,12 +17,12 @@ def test_boolean(self):
         self.assertEqual(_boolean("nee"), False)
         self.assertEqual(_boolean("false"), False)
         self.assertEqual(_boolean(b"false"), False)
-        self.assertEqual(_boolean(u"false"), False)
+        self.assertEqual(_boolean("false"), False)
```

Comment:
Can you remove this and a few other lines like it please?  They are duplicate now, after the remove of the unicode marker.  Originally I explicitly added a test for "x", one for b"x" and one for u"x" to see if Python 2 and 3 handled all three cases properly.  Now only "x" and b"x" are needed.

Reply:
I looked for similar and lines and dups (due to automatic `u""` removal by pyupgrade) - so, i think I found 'em all.